git/services/gitRepoInfo.js

```diff
@@ -5,7 +5,7 @@
  * @return {Object} An object containing the github owner and repository name
  */
 module.exports = function gitRepoInfo(packageInfo) {
-  var GITURL_REGEX = /^https?:\/\/[^\/]+\/([^\/]+)\/(.+).git$/;
+  var GITURL_REGEX = /^[git\+h|h]+ttps:\/\/github.com\/([^\/]+)\/(.+).git$/;
```

Comment:
The regex is more restrictive, before `http://foo/bar/baz.git` was working, it does not work now.